### PR TITLE
[Feature] Add metrics commands and support display in AsciiGraph style

### PR DESCRIPTION
**Pull request by kezhenxu94**
```shell
skywalking-cli $ ./bin/swctl --display=graph metrics global p99 --start '2019-11-18 2300' --end '2019-11-18 2322' 
 3080 ┤╭╮╭──╮ ╭─╮        ╭─╮  
 3079 ┤│││  │ │ │        │ │  
 3078 ┤│││  │ │ │        │ │  
 3077 ┤│││  │ │ │        │ │  
 3076 ┤│││  │ │ │        │ │  
 3075 ┤│││  │ │ │        │ │  
 3074 ┤│││  │ │ │        │ │  
 3073 ┤│││  │ │ │        │ │  
 3072 ┤│││  │ │ │        │ │  
 3071 ┤│││  │ │ │        │ │  
 3070 ┤│││  │ │ ╰───╮ ╭╮╭╯ ╰─ 
 3069 ┤│││  │ │     │ │││     
 3068 ┤│││  │ │     │ │││     
 3067 ┤│││  │ │     │ │││     
 3066 ┤│││  │ │     │ │││     
 3065 ┤│││  │ │     │ │││     
 3064 ┤│││  │ │     │ │││     
 3063 ┤│││  │ │     │ │││     
 3062 ┤│││  │ │     │ │││     
 3061 ┤│││  │ │     │ │││     
 3060 ┼╯╰╯  ╰─╯     ╰─╯╰╯     

skywalking-cli $ ./bin/swctl --display=graph service ls --start '2019-11-18 2300' --end '2019-11-18 2322' 
FATAL type of []schema.Service is not supported to display as graph 
```

### CODE REVIEW

**Comment by wu-sheng** [comment]
Why doesn't it have an x-axis?

**Comment by kezhenxu94** [comment]
> Why doesn't it have an x-axis?

Oops, didn't notice this, will check

**Comment by wu-sheng** [comment]
> --display=graph metrics global p99

Another concern about this, p99 is just a thing defined in OAL scrip, nothing more. If you set this in CLI codes, it is pretty dangerous too me. In my mind and existing design, https://github.com/apache/skywalking/issues/3758, it should matched to graphql, such as `--metrics-linear`, `--metrics-value`.
When you query by `--metrics-linear`, it is a linear graph visualization. When you do `--metrics-value`, it is a single value. By following this, we could have heatmap, top list, even trace and alarm in the CLI.

The way you are doing today is leading to a very limited design.

**Comment by kezhenxu94** [comment]
now it works like this

```shell
skywalking-cli (feature/metrics *+$%) -> make clean && make && ./bin/swctl --debug --display=graph metrics --name=all_p50
```

image

**Comment by wu-sheng** [comment]
1. `metrics` is not an accurate name, I think. I think `linear-metrics` is. Because what do you call when you query single value or heatmap?
1. What happens if I use `--display=yaml`? Is it still working like normal YAML format?

**Comment by wu-sheng** [comment]
Notice, the graph display should be mapping the metrics type automatically, linear or heatmap at least. Don't need to set the type. And single value query and topN query should be not available for `--display =graph`.

**Comment by kezhenxu94** [comment]
> * `metrics` is not an accurate name, I think. I think `linear-metrics` is. Because what do you call when you query single value or heatmap?

make sense to me

> * What happens if I use `--display=yaml`? Is it still working like normal YAML format?

the key and values are `time` and `metrics value` respectively

```shell
skywalking-cli (feature/metrics *$) -> make clean && make && ./bin/swctl --debug --display=yaml metrics --name=all_p95
2019-11-24 1333: 2130
2019-11-24 1334: 2130
2019-11-24 1335: 2130
2019-11-24 1336: 2120
2019-11-24 1337: 2130
2019-11-24 1338: 2140
2019-11-24 1339: 2140
2019-11-24 1340: 2130
2019-11-24 1341: 3100
2019-11-24 1342: 2130
2019-11-24 1343: 2410
2019-11-24 1344: 2120
2019-11-24 1345: 2130
2019-11-24 1346: 2120
2019-11-24 1347: 2130
2019-11-24 1348: 2130
2019-11-24 1349: 3110
2019-11-24 1350: 2130
2019-11-24 1351: 2280
2019-11-24 1352: 2290
2019-11-24 1353: 3100
2019-11-24 1354: 2170
2019-11-24 1355: 3060
2019-11-24 1356: 2080
2019-11-24 1357: 2090
2019-11-24 1358: 3060
2019-11-24 1359: 2100
2019-11-24 1400: 2190
2019-11-24 1401: 2160
2019-11-24 1402: 2130
2019-11-24 1403: 2120
skywalking-cli (feature/metrics $) -> 
```

> Notice, the graph display should be mapping the metrics type automatically, linear or heatmap at least. Don't need to set the type. 

I'm confused why it should be graph display style by default, are there any differences between metrics query and services/instance query?

> And single value query and topN query should be not available for --display =graph.

single value and topN are not supported yet

**Comment by wu-sheng** [comment]
>I'm confused why it should be graph display style by default, are there any differences between metrics query and services/instance query?

I don't mean default. I mean, only linear-metrics could show like this. So you could explicitly say this is linear-metrics query, such as 
```
./bin/swctl --debug --display=graph linear-metrics --name=all_p50
```

**Comment by wu-sheng** [comment]
> single value and topN are not supported yet

Yes. And we could not support them in 0.1.0, right? My point is about subcommand name, like my above comment.

**Comment by wu-sheng** [comment]
Waiting for your `linear-metrics` command update.

File: README.md [hunk 1]
```diff
@@ -114,6 +114,16 @@ and it also has some options and third-level commands.
 | `--start` | See [Common options](#common-options) | See [Common options](#common-options) |
 | `--end` | See [Common options](#common-options) | See [Common options](#common-options) |
 
+### `metrics` second-level command
+`metrics` second-level command is an entrance for all operations related to metrics,
```
Comment (wu-sheng) [inline README.md]: @kezhenxu94 Document change is missing, please recheck whether there are other places.

**Pull request merged**